packages/swap/src/routes/v2/swap.ts

```diff
@@ -239,6 +240,7 @@ router.get(
             swapDepositChannel?.failedBoosts.at(0)?.failedAtBlockIndex ?? undefined,
         },
       }),
+      lastStatechainUpdateAt: await getLastChainTrackingUpdateTimestamp(),
```

Comment:
why do we need to add it here in addition to the headers?

Reply:
missed the conversation in discord. makes sense. but is it possible that the sdk reads it from the header and put it in the final object?

Reply:
Its a good idea but it comes with 2 disadvantages:
- Users calling the API endpoints directly (without using the SDK) will only have the headers to utilize
- Implicit knowledge required in the sdk methods to understand why we are checking for the headers there